shimmer/develop/what-is-shimmer/introduction.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Become a Node Operator
```

Comment:
What's doing on with this titles? :trollface:

Reply:
copy pasted the structure, since docusaurus only allows 1 h1 I must have missed them